docs/compilers/CSharp/Compiler Breaking Changes - post VS2017.md

```diff
@@ -79,3 +79,19 @@ Example: `Func<int> f = default(TypedReference).GetHashCode; // new error CS0123
 - https://github.com/dotnet/roslyn/pull/23416 Before Visual Studio 2017 version 15.6 (Roslyn version 2.8) the compiler accepted `__arglist(...)` expressions with void-typed arguments. For instance, `__arglist(Console.WriteLine())`. But such program would fail at runtime. In Visual Studio 2017 version 15.6, this causes a compile-time error.
 
 - https://github.com/dotnet/roslyn/pull/24023 In Visual Studio 2017 version 15.6, Microsoft.CodeAnalysis.CSharp.Syntax.CrefParameterSyntax constructor and Update(), the parameter refOrOutKeyword was renamed to refKindKeyword (source breaking change if you're using named arguments).
+
+- Visual Studio 2017 15.0-15.5 shipped with a bug around definite assignment of local functions that did not produce definite assignment errors when an uncalled local function contains a nested lambda which captures a variable. For example:
```

Comment:
>which captures a variable [](start = 201, length = 25)

Is the wording here accurate? It doesn't look like the lambda in example captures a variable, it declares one and uses its value before the variable is assigned.